cloudhub/modules/ROOT/pages/ch-ch2-migration-configuration.adoc

```diff
@@ -0,0 +1,536 @@
+
+|CloudHub has limited control over log forwarding.
+a|If you deselect *Forward application logs to Anypoint Platform* within the app configuration in Runtime Manager:
+
+* CloudHub 2.0 doesn’t forward application logs to Runtime Manager or Anypoint Monitoring.
+* The custom log appenders configured in your application continue to function normally (excluding file-based appenders).
+
+This provides more granular control over platform log forwarding.
+
+
+| For more information, see https://help.salesforce.com/s/articleView?id=001117963&type=1[CloudHub 1.0 Logging FAQ]
+| For more information, see https://help.salesforce.com/s/articleView?id=001119527&type=1[CloudHub 2.0 Logging FAQ]
+|===
+
+====
+
+.Deployment Configuration
+[%collapsible,opts=hide-details]
+====
+
+[cols="1,1",options="header"]
+|===
+|CloudHub
+|CloudHub 2.0 
+
+|Use maven and `pom.xml` configuration to deploy applications to CloudHub. See xref:mule-runtime::deploy-to-cloudhub.adoc[].
+a|App deployment with maven and `pom.xml` configuration takes two steps:
+
+. Publish the artifact to Anypoint Exchange (`mvn clean deploy`).
+. Deploy the artifact to CloudHub 2.0 (`mvn clean deploy`, `-DmuleDeploy`).
+
+For more information, see xref:mule-runtime::deploy-to-cloudhub-2.adoc[].
+
+
+|The maximum size of a Mule application (ZIP or JAR) file is 200 MB.
+|The maximum size of a Mule application JAR file is 350 MB.
+
+|Names are unique per control plane.
+a|Names are unique per private space. CloudHub 2.0 adds a six-character unique ID to the application name that you specify in the public endpoint URL: `Myapp-uniq-id.shard.region.cloudhub.io`
+
+* `uniq-id`: is a six-digit value appended to the app name to ensure uniqueness.
+* `shard`: is a six-digit value associated with the space (private or shared) the app is deployed to.
+
+|The DLB supports a maximum HTTP POST (payload) size of 200 MB.
+|The maximum HTTP request size is 1 GB.
+|===
+
+====
+
+.Mule Runtime Versions
+[%collapsible,opts=hide-details,width=100%]
+====
+
+[cols="1,1",options="header,spread,autowidth"]
+|===
+|CloudHub
+|CloudHub 2.0 
+
+|Mule runtime 3.9.x and later.
+|Mule runtime 4.3.0 and later.
+|===
+
+====
+
+.vCore Sizes
+[%collapsible,opts=hide-details,width=100%]
+====
+
+[cols="1,1",options="header,spread,autowidth"]
+|===
+|CloudHub
+|CloudHub 2.0 
+
+|See xref:cloudhub::cloudhub-architecture#cloudhub-workers[CloudHub Workers].
+|See xref:cloudhub-2::ch2-architecture#cloudhub-2-replicas[CloudHub 2.0 Replicas].
+|===
+
+CloudHub 2.0 provides a more granular resource capacity for applications. You can assign specific vCore increments (including 0.5, 1.5, 2.5, 3, 3.5, and 4). This granularity helps match processing needs precisely and avoids forcing a jump from 0.2 to 1 vCores. 
+Replicas with fewer than 1.0 vCores offer limited CPU and I/O suitable for smaller workloads, while those with one or more vCores provide performance consistency.
+====
+
+.Worker Vs. Replica
```

Comment:
I wasn't able to find definitive guidance, but wonder if using Vs. spelled out as Versus would be better for L10N? NotebookLM said it finds internal versions of this, and I do see it in docs.mulesoft.com as vs. (lower case) in titles. I'd suggest using lowercase for consistency if you keep it abbreviated.